### PR TITLE
Element news item

### PR DESCRIPTION
**Pull request by eliotwrobson**
See title. Adds news item detailing new element features. Depends on #7005 (since the screenshots were taken using that feature).

### CODE REVIEW

File: news_items/041_element_question_writing/index.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+</style>
+
+<p>
+  Various new features and quality of life improvements for question creation
+  have been recently deployed. These include new elements, additional features for
+  existing elements, and more utility code to improve the experience of
+  writing custom questions.
+</p>
+<hr>
+<!--https://github.com/PrairieLearn/PrairieLearn/commits?author=eliotwrobson-->
```
Comment (nwalters512) [inline news_items/041_element_question_writing/index.html:31]: Remove this?

File: news_items/041_element_question_writing/index.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    </p>
+  </div>
+</div>
+<hr>
+<h3>Utility Code Improvements</h3>
+<p>
+  There have been substantial improvements to utility code available to instructors in the
+  <code>prairielearn.py</code> Python library, including greater support for JSON serialization
+  and more utility functions usable in custom questions. These changes have primarily been to facilitate writing
+  custom questions and support the new elements and features discussed above. This has all been detailed in
```
Comment (nwalters512) [inline news_items/041_element_question_writing/index.html:153]: I see the phrase "custom question" used throughout this news items. That's not a phrase we use elsewhere; what are you specifically referring to there? If I'm understanding correctly, you should just use "question".

File: news_items/041_element_question_writing/index.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
+<style>
```
Comment (nwalters512) [inline news_items/041_element_question_writing/index.html:1]: Tangential thought: we should really update the news item machinery to support writing them in Markdown.

Comment (eliotwrobson) [reply news_items/041_element_question_writing/index.html:1]: I had the same thought while writing this 😅

File: news_items/041_element_question_writing/index.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  have been recently deployed. These include new elements, additional features for
+  existing elements, and more utility code to improve the experience of
+  writing custom questions.
+</p>
+<hr>
+<!--https://github.com/PrairieLearn/PrairieLearn/commits?author=eliotwrobson-->
+<h3>New Elements</h3>
+<br>
+<div>
+  <div class="card">
```
[the remark this answers is not in the document]
Comment (eliotwrobson) [reply news_items/041_element_question_writing/index.html:35]: Sounds good, I'll use a few divs or something to try and split this up.

[the remark this answers is not in the document]
Comment (nwalters512) [reply news_items/041_element_question_writing/index.html]: Even the divs are unnecessary (i.e. Markdown won't spit them out), but I don't mind that they're there!

I think the things you've covered are sufficient 👍

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**